internal/models/view.go

```diff
@@ -0,0 +1,150 @@
+package models
+
+import (
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type Noteable interface {
+	GetName() string
+	GetType() TypeNote
+	GetId() uuid.UUID
+	Print() string
+}
+
+type TypeNote string
+
+func (t TypeNote) String() string {
+	return string(t)
+}
+
+const (
+	CARD       TypeNote = "bank card"
+	CREDENTIAL TypeNote = "credential"
+	TEXT       TypeNote = "text"
+	BINARY     TypeNote = "binary"
+)
+
+type BaseNote struct {
+	Id         uuid.UUID `json:"id"`
+	NameRecord string    `json:"name_record"`
+	Created    int64     `json:"created"`
+	Type       TypeNote  `json:"type"`
+	MetaInfo   []string  `json:"meta_info,omitempty"`
+}
+
+type CredentialNote struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+	BaseNote `json:"data"`
+}
+
+func (cn CredentialNote) Print() string {
+	var str string
+	str += "Note: " + cn.NameRecord + "\n"
+	str += "Username: " + cn.Username + "\n"
+	str += "Password: " + cn.Password + "\n"
+	str += "Additional information: " + strings.Join(cn.MetaInfo, "; ") + "\n"
+	str += "Created: " + time.Unix(cn.Created, 0).Format(time.RFC822) + "\n"
+	return str
+}
+
+func (cn CredentialNote) GetName() string {
+	return cn.NameRecord
+}
+
+func (cn CredentialNote) GetType() TypeNote {
+	return CREDENTIAL
+}
+
+func (cn CredentialNote) GetId() uuid.UUID {
```

Comment:
```suggestion
func (cn CredentialNote) GetID() uuid.UUID {
```

ну вот так в го принято